src/views.ts

```diff
@@ -263,6 +267,11 @@ export function addOverworldEventListeners(overworld: Overworld) {
           }
         }
       },
+      {
+        target: elChatinput,
+        event: 'keypress',
```

Comment:
If you wrap the input in a <form/> and give it `type="submit"` you can use the 'submit' event which will listen to Enter but not process other commands.  Then just remember to call `e.preventDefault()` in the event handler.  The benefit of this is that it wont listen to every keystroke.
Further reading: https://stackoverflow.com/a/27808062/4418836